src/test/java/JavAI/StorageTest.java

```diff
@@ -0,0 +1,55 @@
+package javai;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
```

Comment:
Could perhaps group the import statements and separate them out with newlines? Like all the junit together, and the utils together.